.github/AL-Go-Settings.json

```diff
@@ -16,16 +16,10 @@
       ],
       "settings": {
         "preprocessorSymbols": [
-          "CLEAN17",
-          "CLEAN18",
-          "CLEAN19",
-          "CLEAN20",
-          "CLEAN21",
-          "CLEAN22",
-          "CLEAN23",
           "CLEAN24",
```

Comment:
I think CLEAN23 is technically still allowed (even though people shouldn't be using it of course)

Reply:
I added it back. Only because we will be moving other apps as well.